core/src/main/java/com/sequenceiq/cloudbreak/core/flow/service/AmbariClusterFacade.java

```diff
@@ -160,17 +175,16 @@ public FlowContext startCluster(FlowContext context) throws CloudbreakException
     @Override
     public FlowContext stopCluster(FlowContext context) throws CloudbreakException {
         StackStatusUpdateContext statusUpdateContext = (StackStatusUpdateContext) context;
+        updateClusterStatus(statusUpdateContext.getStackId(), STOP_IN_PROGRESS, "Services are stopping.");
```

Comment:
to stay consistent along the use cases please set the stack status to update_in_progres during cluster stop
(we do so when we are creating the cluster) - let this be a kind of signal that something is happening on the stack